codeception.dist.yml

```diff
@@ -31,7 +31,7 @@ modules:
         - /var/www/magento/pub/media
         - /var/www/magento/var
         - /var/www/magento/app/etc
-      printOutput: false
+      printOutput: true
```

Comment:
Is it a good idea to change this value? Logs in Travis can be very huge.

Reply:
Forgotten debug stmt, removed